src/ansys/fluent/core/workflow.py

```diff
@@ -1258,20 +1258,21 @@ def compound_child(self, name: str):
 
 def _makeTask(command_source, name: str) -> BaseTask:
     task = command_source._workflow.TaskObject[name]
-    task_type = task.TaskType()
     kinds = {
+        None: BaseTask,
```

Comment:
Only issue here is that even when there might be a possibility that we can know the task types, we are going ahead with BaseTask. However, this is done to support old journals, so this should be fine.